google_authorized_keys/main.go

```diff
@@ -20,23 +20,24 @@ import (
 	"encoding/json"
 	"fmt"
 	"io"
-	"net/http"
 	"os"
 	"runtime"
 	"strconv"
 	"strings"
 	"time"
 
+	"github.com/GoogleCloudPlatform/guest-agent/metadata"
 	"github.com/GoogleCloudPlatform/guest-agent/utils"
 	"github.com/GoogleCloudPlatform/guest-logging-go/logger"
 )
 
-var (
-	programName       = "GoogleAuthorizedKeysCommand"
-	metadataURL       = "http://169.254.169.254/computeMetadata/v1/"
-	metadataRecursive = "/?recursive=true&alt=json"
-	defaultTimeout    = 2 * time.Second
-)
+const programName = "GoogleAuthorizedKeysCommand"
```

Comment:
I don't know why we were originally using `GoogleAuthorizedKeysCommand` here as it should be the binary name i.e. `path.Base(os.Args[0])`.

Reply:
Done